kitsune/products/jinja2/products/mobile/product.html

```diff
@@ -16,15 +16,17 @@
   <ul id="topics">
     {% for topic in topics %}
       <li>
-        <a href="{{ url('products.documents', product_slug=product.slug, topic_slug=topic.slug) }}" class="cf">
+        <a href="{{ url('products.documents', product_slug=product.slug, topic_slug=topic.slug) }}" class="cf"
+        data-event-category="link click" data-event-action="topic" data-event-label="{{ topic.title }}">>
           <img src="{{ STATIC_URL }}sumo/img/blank.png" class="topic-sprite topic-{{ topic.slug }}" alt="">
           {{ pgettext('DB: products.Topic.title', topic.title) }}
         </a>
       </li>
       {% if loop.last %}
           <li>
             <a href="{{ url('wiki.document', 'get-community-support') }}" class="cf">
-              <img src="{{ STATIC_URL }}sumo/img/blank.png" class="topic-sprite topic-get-community-support" alt="">
+              <img src="{{ STATIC_URL }}sumo/img/blank.png" class="topic-sprite topic-get-community-support"
+              alt="" data-event-category="link click" data-event-action="topic" data-event-label="Get community support">>
```

Comment:
Should these `data-` attributes be on the `<a>` instead of the `<img>`?

Reply:
Yes!